src/schema/v2/conversation/index.ts

```diff
@@ -339,6 +341,65 @@ export const ConversationType = new GraphQLObjectType<any, ResolverContext>({
         }
       },
     },
+    collectorInterestsConnection: {
+      type: UserInterestConnection,
+      args: pageable({}),
+      resolve: async (
+        { from_id, to_id },
+        args,
+        {
+          partnerCollectorProfileUserInterestsLoader,
+          partnerCollectorProfileLoader,
+        }
+      ) => {
+        if (
+          !partnerCollectorProfileUserInterestsLoader ||
+          !partnerCollectorProfileLoader ||
+          !from_id ||
+          !to_id
+        ) {
+          return null
+        }
+
+        try {
+          const { page, size, offset } = convertConnectionArgsToGravityArgs(
+            args
+          )
+          const data = await partnerCollectorProfileLoader({
```

Comment:
Leider, it's the downside of this, one extra request to get the `collectorProfileId` 😢